app/views/explorer_stake_accounts/_stats_row.html.erb

```diff
@@ -0,0 +1,64 @@
+<% if @stake_history.any? %>
+  <div class="row">
+    <div class="col-lg-8 mb-4">
+      <div class="card h-100">
+        <div class="card-content pb-3">
+          <h2 class="h4 card-heading">Stake Stats</h2>
+          <%
+            average_active_stake = @stake_history.map do |stake| 
+              stake.average_active_stake ? lamports_to_sol(stake.average_active_stake).round(2) : 0
+            end
+            active_stake_vector = @stake_history.map do |stake| 
+              stake.active_stake ? lamports_to_sol(stake.active_stake).round(2) : 0
+            end
+            epochs = @stake_history.pluck(:epoch)
+          %>
+          <%= render 'explorer_stake_accounts/stats_chart',
+                    chart_name: "stake-stats-chart",
+                    epochs: epochs,
+                    average_active_stake: average_active_stake,
+                    active_stake_vector: active_stake_vector
+          %>
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-4 mb-4">
+      <div class="card h-100">
+        <div class="card-content pb-3">
+          <h2 class="h4 card-heading">Last Epoch</h2>
+        </div>
+        <table class='table table-block mb-0'>
+          <tbody>
+            <tr>
+              <td><strong>Delegated Stake:</strong></td>
+              <td><%= number_with_delimiter lamports_to_sol(@stake_history.last.delegated_stake).round(2) %> SOL</td>
+            </tr>
+            <tr>
+              <td><strong>Active Stake:</strong></td>
+              <td><%= number_with_delimiter lamports_to_sol(@stake_history.last.active_stake).round(2) %> SOL</td>
+            </tr>
+            <tr>
+              <td><strong>Avg. Active Stake:</strong></td>
+              <td><%= number_with_delimiter lamports_to_sol(@stake_history.last.average_active_stake).round(2) %> SOL</td>
```

Comment:
Throws an error if average_active_stake is nil:
https://appsignal.com/first-movers-advantage-llc/sites/6043bed214ad6641f161a691/exceptions/incidents/313/samples/6043bed214ad6641f161a691-843910321233785725517084568601